src/server/engine_shard_set.cc

```diff
@@ -715,9 +715,10 @@ void EngineShard::CacheStats() {
     DbTable* table = db_slice.GetDBTable(i);
     if (table) {
       entries += table->prime.size();
-      table_memory += (table->prime.mem_usage() + table->expire.mem_usage());
+      table_memory += table->table_memory();
     }
   }
+  DCHECK_EQ(table_memory, db_slice.table_memory());
```

Comment:
This is the check that passed the CI

Reply:
I don't understand what you mean in this comment :|

Reply:
I have not used ` db_slice.table_memory()` anywhere, I just compute it in this PR. I also kept the old way of computing  table_memory (in this section). This DCHECK make sure that the new approach is consistent with the old one.

Reply:
Basically, it's a preparation PR that we can leave running for a few days, seeing we do not find any issues with the new approach before deleting the old code.

Reply:
oh I see! that's a great idea.